fdbserver/workloads/ApiCorrectness.actor.cpp

```diff
@@ -114,6 +132,11 @@ struct ApiCorrectnessWorkload : ApiWorkload {
 		int maxTransactionBytes = getOption(options, "maxTransactionBytes"_sr, 500000);
 		maxKeysPerTransaction = std::max(1, maxTransactionBytes / (maxValueLength + maxLongKeyLength));
 
+		validateEncryptionAtRest =
+		    g_simulator->isSimulated()
```

Comment:
```suggestion
		    g_network->isSimulated()
```

Reply:
@sfc-gh-ahusain Accessing `g_simulator` is actually a segfault outside of simulation.

Reply:
Yup it is causing my CI builds to fail. Updated them, thanks @sfc-gh-nwijetunga for pointing out. For some reason I thought, `g_simulator->isSimulated()` will work, which ironically doesn't :)